client/state/themes/test/actions.js

```diff
@@ -805,47 +814,36 @@ describe( 'actions', () => {
 			} );
 		} );
 
-		test( 'should dispatch status update', () => {
-			return pollThemeTransferStatus(
-				siteId,
-				3,
-				'themes',
-				20
-			)( spy ).then( () => {
-				// Two 'progress' then a 'complete'
-				expect( spy ).to.have.callCount( 4 );
-				expect( spy ).to.have.been.calledWith( {
-					type: THEME_TRANSFER_STATUS_RECEIVE,
-					siteId: siteId,
-					transferId: 3,
-					status: 'progress',
-					message: 'in progress',
-					themeId: undefined,
-				} );
-				expect( spy ).to.have.been.calledWith( {
-					type: THEME_TRANSFER_STATUS_RECEIVE,
-					siteId: siteId,
-					transferId: 3,
-					status: 'complete',
-					message: 'all done',
-					themeId: 'mood',
-				} );
+		test( 'should dispatch status update', async () => {
```

Comment:
**Note:**
These test changes are unrelated to this PR. They're meant to fix failing eslint rules that were making teamCity CI angry.